device/ragile/x86_64-ragile_ra-b6920-4s-r0/plugins/ssd_util.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, diskdev):
         self.temperature = NOT_AVAILABLE
         self.health = NOT_AVAILABLE
 
-        self.ssd_info = self._execute_shell(INNODISK.format(diskdev))
+        self.ssd_info = self._execute_shell(INNODISK.append(str(diskdev)))
```

Comment:
>str [](http://example.com/codeflow?start=60&length=3)

`str` is not needed, because L21 already checked it is a `str`.

Reply:
I updated, thank you.